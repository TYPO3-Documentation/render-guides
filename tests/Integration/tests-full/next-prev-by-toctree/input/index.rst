==========
Root index
==========

Lorem Ipsum Dolor.

..  toctree::

    one
    two
    three/index
    four
