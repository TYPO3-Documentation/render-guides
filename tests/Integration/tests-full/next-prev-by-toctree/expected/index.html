<!DOCTYPE html>
<html class="no-js" lang="en">
<head>
    <title>Root index documentation</title>

    
<meta charset="utf-8">
<meta content="width=device-width, initial-scale=1.0" name="viewport">
<meta content="phpdocumentor/guides" name="generator">
<meta content="" name="docsearch:name">
<meta content="" name="docsearch:package_type">
<meta content="" name="docsearch:release">
<meta content="" name="docsearch:version">
<meta content="2023-01-01T12:00:00+00:00" name="docsearch:modified">
<meta content="2023-01-01T12:00:00+00:00" name="dc.modified">
<meta content="2023-01-01T12:00:00+00:00" property="article:modified_time">
    <link href="http://purl.org/dc/elements/1.1/" rel="schema.dc">
<link href="_resources/css/theme.css" rel="stylesheet">
<link href="https://docs.typo3.org/search/" rel="search" title="Search">
<script src="https://typo3.azureedge.net/typo3infrastructure/universe/dist/webcomponents-loader.js"></script>
<script src="https://typo3.azureedge.net/typo3infrastructure/universe/dist/typo3-universe.js" type="module"></script>

                        <link href="one.html" rel="next" title="One"/>
            <link href="#" rel="top" title="Root index"/>
    </head>
<body cz-shortcut-listen="true">
<div class="page">
    
<header>
    <div class="page-topbar">
        <div class="page-topbar-inner">
            <typo3-universe active="documentation">
                <div style="display: block; height: 44px; background-color: #313131;"></div>
            </typo3-universe>
        </div>
    </div>
    <div class="page-header">
        <div class="page-header-inner">
            <a class="logo" href="https://docs.typo3.org/" title="TYPO3 Documentation">
                <img alt="TYPO3 Logo" class="logo-image" src="_resources/img/typo3-logo.svg" width="484" height="130">
            </a>
        </div>
    </div>
</header>
    <main class="page-main">
        <div class="page-main-inner">
            <div class="page-main-navigation">
                <nav>
                    <input class="toc-checkbox" id="toggleToc" type="checkbox">

                    
<div class="toc-header">
    <div class="toc-title">
        <a class="toc-title-project" href="#"></a>
            </div>
    <div class="toc-actions">
        <label class="toc-toggle" for="toggleToc">
            Menu
        </label>
    </div>
</div>
                    <div class="toc-collapse">
                                                <div aria-label="main navigation" class="toc" role="navigation">
                                <div aria-label="Main navigation" class="main_menu" role="navigation">
        
    <ul class="menu-level-1">
    <li class="">
    <a href="one.html">
        One
    </a></li>
    <li class="">
    <a href="two.html">
        Two
    </a></li>
    <li class="">
    <a href="three/index.html">
        Three point something
    </a>        <ul class="menu-level-1"><li class="">
    <a href="three/three.html">
        Three
    </a></li>
<li class="">
    <a href="three/threepointfive.html">
        3.5
    </a></li>
</ul></li>
    <li class="">
    <a href="four.html">
        Four
    </a></li>
    </ul>


    </div>
                        </div>
                    </div>
                </nav>
            </div>
            <div class="page-main-content">
                <div class="rst-content">                        <nav aria-label="breadcrumbs navigation" class="breadcrumb-bar" role="navigation">
        
<ol class="breadcrumb">
                <li aria-current="page"  class="breadcrumb-item  active">Root index</li>
        </ol>

        <div class="breadcrumb-additions">    <a class="btn btn-sm btn-light" href="_sources/index.rst.txt" rel="nofollow noopener" target="_blank">
        <span class="btn-icon"><span class="fas fa-code"></span></span>
        <span class="btn-text">View source</span>
    </a>
        <a class="btn btn-sm btn-light" href="https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingDocsOfficial/GithubMethod.html" id="btnHowToEdit" rel="nofollow noopener" target="_blank">
        <span class="btn-icon"><span class="fas fa-info-circle"></span></span>
        <span class="btn-text">How to edit</span>
    </a>
    </div>
    </nav>
                    
<article class="document" itemscope="itemscope" itemtype="http://schema.org/Article" role="main">
    <div itemprop="articleBody">
        <!-- content start -->
                <section class="section" id="root-index">
            <h1>Root index<a class="headerlink" href="#root-index" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">Â¶</a></h1>

            <p>Lorem Ipsum Dolor.</p>
            <div class="toctree-wrapper compound">
    <ul class="menu-level">
    <li class="toc-item"><a href="one.html#one">One</a></li>

    <li class="toc-item"><a href="two.html#two">Two</a></li>

    <li class="toc-item"><a href="three/index.html#three-point-something">Three point something</a>        <ul class="menu-level-1">
            <li class="toc-item"><a href="three/three.html#three">Three</a></li>

            <li class="toc-item"><a href="three/threepointfive.html#3-5">3.5</a></li>

                    </ul></li>

    <li class="toc-item"><a href="four.html#four">Four</a></li>

    </ul>
</div>

    </section>

        <!-- content end -->
    </div>
</article>
                        
            <nav aria-label="Page navigation">
            <ul class="pagination justify-content-center"><li class="page-item">
                            <a class="page-link" href="one.html"
                               title="Accesskey Alt(+Shift)+n">
                                Next
                            </a>
                        </li>
                    </ul>
        </nav>
                    </div>
            </div>
        </div>
    </main>

    <div class="modal fade" id="linkReferenceModal" tabindex="-1" aria-labelledby="linkReferenceModalLabel"
    aria-hidden="true" data-current-filename="index">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h5 class="modal-title" id="linkReferenceModalLabel">Reference to the headline</h5>
                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
            </div>
            <div class="modal-body">
                <div class="alert alert-success d-none" id="permalink-alert-success" role="alert"></div>
                <div class="mb-3">
                    <label for="permalink-uri" class="col-form-label">Permalink</label>
                    <div class="input-group">
                        <input class="form-control code" id="permalink-uri" readonly>
                        <button type="button" class="btn btn-outline-secondary copy-button" data-target="permalink-uri"><i class="far fa-clone"></i></button>
                    </div>
                    <p>Copy and freely share the link</p>
                </div>
                <div class="mb-3">
                    <div class="alert alert-warning alert-permalink-rst" role="alert">This link target has no permanent anchor assigned.The link below can be used, but is prone to change if the page gets moved.
                    </div>
                    <label for="permalink-rst" class="col-form-label">reStructuredText (reST):</label>
                    <div class="input-group">
                        <textarea class="form-control code" id="permalink-rst" readonly></textarea>
                        <button type="button" class="btn btn-outline-secondary copy-button" data-target="permalink-rst"><i class="far fa-clone"></i></button>
                    </div>
                    <p>Copy this link into your TYPO3 manual. </p>
                </div>
                <div class="mb-3">
                    <label for="permalink-html" class="col-form-label">HTML:</label>
                    <div class="input-group">
                        <textarea class="form-control code" id="permalink-html" readonly></textarea>
                        <button type="button" class="btn btn-outline-secondary copy-button" data-target="permalink-html"><i class="far fa-clone"></i></button>
                    </div>
                </div>
            </div>
            <div class="modal-footer">
                <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
            </div>
        </div>
    </div>
</div>
</div>


<script src="_resources/js/popper.min.js"></script>
<script src="_resources/js/bootstrap.min.js"></script>
<script src="_resources/js/theme.min.js"></script>

<script id="R9wBKTwzv" src="https://app.usercentrics.eu/latest/main.js"></script>
<script data-usercentrics="Matomo" src="/js/piwik.js" type="text/plain"></script>

<script type="text/plain" data-usercentrics="Google Tag Manager">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-MKNXD8Q');</script>
</body>
</html>
